en/index.md

```diff
@@ -378,27 +378,31 @@ This will work, until such a time as whoever owns the image closes their account
 ## Words and phrases to avoid
 
 Try to avoid jargon, long-winded phrases and words with negative
-connotations. Steer clear of the following:
-
- - *Allow* - This suggests that we are in a position of power,
-permitting users or customers to conduct certain activities.
- - *The ability to* – Use 'We can' instead of 'We have the ability to'
- - *Is able to* – Use 'Ubuntu can' instead of 'Ubuntu is able to'
- - *Not only...but also...*
- - *Eliminate*
- - *Execute*
- - *Terminate*
- - *Kill*
- - *Disruptive*
- - *Explosive*
- - *Leverage*
- - *Ecosystem*
- - *Going forward*
- - *In order to*
- - *Form factor*
- - *Use case*
- - *End user* – Use 'user' instead
- - *Linux for human beings*
+connotations:
+
+| Avoided              | Motivation              | Substitution examples               |
+|----------------------|-------------------------|-------------------------------------|
+|Allow                 | This suggests that we are in a position of power, permitting users or customers to conduct certain activities.|                          |
+|The ability to        |                         | Use 'We can' instead of 'We have the ability to'                        |
+|Is able to            |                         | Use 'Ubuntu can' instead of 'Ubuntu is able to' |
+|Not only...but also...|                         |                               |
+|Eliminate             |                         |                               |
+|Execute               |                         |                               |
+|Terminate             |                         |                               |
+|Kill                  |                         |                               |
+|Disruptive            |                         |                               |
+|Explosive             |                         |                               |
+|Leverage              |                         |                               |
+|Ecosystem             |                         |                               |
+|Going forward         |                         |                               |
+|In order to           |                         |                               |
+|Form factor           |                         |                               |
+|Use case 	           |                         |                               |
```

Comment:
I really don't understand why 'use case' is listed but that may be a question for the style guide SIG.

Reply:
Same @keirthana . In a non-software context "application" would usually suffice here (the practical application of a concept or tool) but given that it has other prominent meanings in software, and without thinking of a superior alternative, I'm personally OK with "use case".